Flaky zha climate test

<!-- READ THIS FIRST:
  - If you need additional help with this template, please refer to https://www.home-assistant.io/help/reporting_issues/
  - Make sure you are running the latest version of Home Assistant before reporting an issue: https://github.com/home-assistant/core/releases
  - Do not report issues for integrations if you are using custom components or integrations.
  - Provide as many details as possible. Paste logs, configuration samples and code into the backticks.
  DO NOT DELETE ANY TEXT from this template! Otherwise, your issue may be closed without comment.
-->
## The problem
<!-- 
  Describe the issue you are experiencing here to communicate to the
  maintainers. Tell us what you were trying to do and what happened.
-->
- There's a flaky zha climate test on dev branch: `test_set_fan_mode_no_fan`

## Environment
<!--
  Provide details about the versions you are using, which helps us to reproduce
  and find the issue quicker. Version information is found in the
  Home Assistant frontend: Developer tools -> Info.
-->

- Home Assistant Core release with the issue: 0.111.0.dev0
- Last working Home Assistant Core release (if known): 
- Operating environment (Home Assistant/Supervised/Docker/venv): Azure pipelines
- Integration causing this issue: zha
- Link to integration documentation on our website: 

## Problem-relevant `configuration.yaml`
<!--
  An example configuration that caused the problem for you. Fill this out even
  if it seems unimportant to you. Please be sure to remove personal information
  like passwords, private URLs and other credentials.
-->

```yaml

```

## Traceback/Error logs
<!--
  If you come across any trace or error logs, please provide them.
-->

```txt
2020-05-20T22:39:29.2118282Z ##[error]1 test(s) failed, 9244 test(s) collected.
2020-05-20T22:39:29.2146851Z Reading code coverage summary from '/home/vsts/work/1/s/coverage.xml'
2020-05-20T22:39:29.4747965Z Publishing test results to test run '68162'
2020-05-20T22:39:29.4749024Z Test results remaining: 9244. Test run id: 68162
2020-05-20T22:39:29.5834057Z 
2020-05-20T22:39:29.5835138Z =================================== FAILURES ===================================
2020-05-20T22:39:29.5835651Z _______________________ test_set_fan_mode_no_fan[pyloop] _______________________
2020-05-20T22:39:29.5836702Z [gw0] linux -- Python 3.7.0 /__w/1/s/venv/bin/python
2020-05-20T22:39:29.5837035Z 
2020-05-20T22:39:29.5837323Z hass = <homeassistant.core.HomeAssistant object at 0x7f4ef505f748>
2020-05-20T22:39:29.5837791Z device_climate = <homeassistant.components.zha.core.device.ZHADevice object at 0x7f4ef653aac8>
2020-05-20T22:39:29.5838265Z caplog = <_pytest.logging.LogCaptureFixture object at 0x7f4ef64011d0>
2020-05-20T22:39:29.5838892Z 
2020-05-20T22:39:29.5839318Z     async def test_set_fan_mode_no_fan(hass, device_climate, caplog):
2020-05-20T22:39:29.5839739Z         """Test setting fan mode on fun less climate."""
2020-05-20T22:39:29.5840031Z     
2020-05-20T22:39:29.5840341Z         entity_id = await find_entity_id(DOMAIN, device_climate, hass)
2020-05-20T22:39:29.5840637Z     
2020-05-20T22:39:29.5840921Z         with caplog.at_level(logging.DEBUG):
2020-05-20T22:39:29.5841274Z             await hass.services.async_call(
2020-05-20T22:39:29.5841552Z                 DOMAIN,
2020-05-20T22:39:29.5841839Z                 SERVICE_SET_FAN_MODE,
2020-05-20T22:39:29.5842208Z                 {ATTR_ENTITY_ID: entity_id, ATTR_FAN_MODE: FAN_ON},
2020-05-20T22:39:29.5842581Z                 blocking=True,
2020-05-20T22:39:29.5842840Z             )
2020-05-20T22:39:29.5843119Z >       assert "Fan is not supported" in caplog.text
2020-05-20T22:39:29.5844352Z E       AssertionError: assert 'Fan is not supported' in 'DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event call_service[L]: domain=climate, service=set_fan_mode, service_data=entity_id=climate.unk_manufacturer_fakemodel_e769900a_thermostat, fan_mode=on>\n'
2020-05-20T22:39:29.5846069Z E        +  where 'DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event call_service[L]: domain=climate, service=set_fan_mode, service_data=entity_id=climate.unk_manufacturer_fakemodel_e769900a_thermostat, fan_mode=on>\n' = <_pytest.logging.LogCaptureFixture object at 0x7f4ef64011d0>.text
2020-05-20T22:39:29.5846813Z 
2020-05-20T22:39:29.5847095Z tests/components/zha/test_climate.py:1016: AssertionError
2020-05-20T22:39:29.5847737Z ---------------------------- Captured stderr setup -----------------------------
2020-05-20T22:39:29.5848158Z DEBUG:asyncio:Using selector: EpollSelector
2020-05-20T22:39:29.5848519Z DEBUG:zigpy.group:Adding group: 4097, fixture group
2020-05-20T22:39:29.5848950Z INFO:homeassistant.loader:Loaded test_package from custom_components.test_package
2020-05-20T22:39:29.5849403Z INFO:homeassistant.loader:Loaded test from custom_components.test
2020-05-20T22:39:29.5849846Z INFO:homeassistant.loader:Loaded zha from homeassistant.components.zha
2020-05-20T22:39:29.5850239Z INFO:homeassistant.setup:Setting up zha
2020-05-20T22:39:29.5850603Z INFO:homeassistant.setup:Setup of domain zha took 0.0 seconds.
2020-05-20T22:39:29.5851421Z DEBUG:homeassistant.components.zha.core.discovery:Group: fixture group:0x1001 has less than 2 members - skipping entity discovery
2020-05-20T22:39:29.5852138Z DEBUG:homeassistant.core:Bus:Handling <Event device_registry_updated[L]: action=create, device_id=cba5b3729cef47aab2db88369c900597>
2020-05-20T22:39:29.5852760Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=zha, service=permit>
2020-05-20T22:39:29.5853336Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=zha, service=remove>
2020-05-20T22:39:29.5853945Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=zha, service=set_zigbee_cluster_attribute>
2020-05-20T22:39:29.5854579Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=zha, service=issue_zigbee_cluster_command>
2020-05-20T22:39:29.5855222Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=zha, service=issue_zigbee_group_command>
2020-05-20T22:39:29.5855867Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=zha, service=warning_device_squawk>
2020-05-20T22:39:29.5856488Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=zha, service=warning_device_warn>
2020-05-20T22:39:29.5857199Z DEBUG:homeassistant.components.zha.core.gateway:Loading battery powered devices
2020-05-20T22:39:29.5857696Z DEBUG:homeassistant.components.zha.core.gateway:Loading mains powered devices
2020-05-20T22:39:29.5858198Z INFO:homeassistant.loader:Loaded binary_sensor from homeassistant.components.binary_sensor
2020-05-20T22:39:29.5858793Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=zha>
2020-05-20T22:39:29.5859309Z INFO:homeassistant.setup:Setting up binary_sensor
2020-05-20T22:39:29.5859709Z INFO:homeassistant.setup:Setup of domain binary_sensor took 0.0 seconds.
2020-05-20T22:39:29.5860213Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=binary_sensor>
2020-05-20T22:39:29.5860727Z INFO:homeassistant.components.binary_sensor:Setting up binary_sensor.zha
2020-05-20T22:39:29.5861182Z INFO:homeassistant.loader:Loaded climate from homeassistant.components.climate
2020-05-20T22:39:29.5861652Z INFO:homeassistant.loader:Loaded cover from homeassistant.components.cover
2020-05-20T22:39:29.5862149Z INFO:homeassistant.loader:Loaded device_tracker from homeassistant.components.device_tracker
2020-05-20T22:39:29.5862621Z INFO:homeassistant.loader:Loaded fan from homeassistant.components.fan
2020-05-20T22:39:29.5863077Z INFO:homeassistant.loader:Loaded light from homeassistant.components.light
2020-05-20T22:39:29.5863540Z INFO:homeassistant.loader:Loaded lock from homeassistant.components.lock
2020-05-20T22:39:29.5863989Z INFO:homeassistant.loader:Loaded sensor from homeassistant.components.sensor
2020-05-20T22:39:29.5864452Z INFO:homeassistant.loader:Loaded switch from homeassistant.components.switch
2020-05-20T22:39:29.5864860Z INFO:homeassistant.setup:Setting up climate
2020-05-20T22:39:29.5865316Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=climate, service=turn_on>
2020-05-20T22:39:29.5865907Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=climate, service=turn_off>
2020-05-20T22:39:29.5866503Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=climate, service=set_hvac_mode>
2020-05-20T22:39:29.5867100Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=climate, service=set_preset_mode>
2020-05-20T22:39:29.5867707Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=climate, service=set_aux_heat>
2020-05-20T22:39:29.5868326Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=climate, service=set_temperature>
2020-05-20T22:39:29.5868915Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=climate, service=set_humidity>
2020-05-20T22:39:29.5869515Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=climate, service=set_fan_mode>
2020-05-20T22:39:29.5870116Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=climate, service=set_swing_mode>
2020-05-20T22:39:29.5870624Z INFO:homeassistant.setup:Setup of domain climate took 0.0 seconds.
2020-05-20T22:39:29.5871016Z INFO:homeassistant.setup:Setting up cover
2020-05-20T22:39:29.5871480Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=cover, service=open_cover>
2020-05-20T22:39:29.5872053Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=cover, service=close_cover>
2020-05-20T22:39:29.5872677Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=cover, service=set_cover_position>
2020-05-20T22:39:29.5873289Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=cover, service=stop_cover>
2020-05-20T22:39:29.5873855Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=cover, service=toggle>
2020-05-20T22:39:29.5874442Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=cover, service=open_cover_tilt>
2020-05-20T22:39:29.5875049Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=cover, service=close_cover_tilt>
2020-05-20T22:39:29.5875636Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=cover, service=stop_cover_tilt>
2020-05-20T22:39:29.5876252Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=cover, service=set_cover_tilt_position>
2020-05-20T22:39:29.5876878Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=cover, service=toggle_cover_tilt>
2020-05-20T22:39:29.5877485Z INFO:homeassistant.setup:Setup of domain cover took 0.0 seconds.
2020-05-20T22:39:29.5877932Z INFO:homeassistant.loader:Loaded zone from homeassistant.components.zone
2020-05-20T22:39:29.5878330Z INFO:homeassistant.setup:Setting up fan
2020-05-20T22:39:29.5878768Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=fan, service=turn_on>
2020-05-20T22:39:29.5879347Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=fan, service=turn_off>
2020-05-20T22:39:29.5879925Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=fan, service=toggle>
2020-05-20T22:39:29.5880480Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=fan, service=set_speed>
2020-05-20T22:39:29.5881057Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=fan, service=oscillate>
2020-05-20T22:39:29.5881643Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=fan, service=set_direction>
2020-05-20T22:39:29.5882139Z INFO:homeassistant.setup:Setup of domain fan took 0.0 seconds.
2020-05-20T22:39:29.5882520Z INFO:homeassistant.setup:Setting up light
2020-05-20T22:39:29.5882865Z INFO:homeassistant.setup:Setting up lock
2020-05-20T22:39:29.5883305Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=lock, service=unlock>
2020-05-20T22:39:29.5883874Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=lock, service=lock>
2020-05-20T22:39:29.5884441Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=lock, service=open>
2020-05-20T22:39:29.5884918Z INFO:homeassistant.setup:Setup of domain lock took 0.0 seconds.
2020-05-20T22:39:29.5885304Z INFO:homeassistant.setup:Setting up sensor
2020-05-20T22:39:29.5885695Z INFO:homeassistant.setup:Setup of domain sensor took 0.0 seconds.
2020-05-20T22:39:29.5886067Z INFO:homeassistant.setup:Setting up switch
2020-05-20T22:39:29.5886538Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=switch, service=turn_off>
2020-05-20T22:39:29.5887130Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=switch, service=turn_on>
2020-05-20T22:39:29.5887691Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=switch, service=toggle>
2020-05-20T22:39:29.5888195Z INFO:homeassistant.setup:Setup of domain switch took 0.0 seconds.
2020-05-20T22:39:29.5888672Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=climate>
2020-05-20T22:39:29.5889184Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=cover>
2020-05-20T22:39:29.5889714Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=fan>
2020-05-20T22:39:29.5890217Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=lock>
2020-05-20T22:39:29.5890736Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=sensor>
2020-05-20T22:39:29.5891260Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=switch>
2020-05-20T22:39:29.5891722Z INFO:homeassistant.components.climate:Setting up climate.zha
2020-05-20T22:39:29.5892140Z INFO:homeassistant.components.cover:Setting up cover.zha
2020-05-20T22:39:29.5892543Z INFO:homeassistant.components.fan:Setting up fan.zha
2020-05-20T22:39:29.5893014Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=light, service=turn_on>
2020-05-20T22:39:29.5893596Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=light, service=turn_off>
2020-05-20T22:39:29.5894174Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=light, service=toggle>
2020-05-20T22:39:29.5894658Z INFO:homeassistant.setup:Setup of domain light took 0.0 seconds.
2020-05-20T22:39:29.5895071Z INFO:homeassistant.components.lock:Setting up lock.zha
2020-05-20T22:39:29.5895479Z INFO:homeassistant.components.sensor:Setting up sensor.zha
2020-05-20T22:39:29.5895948Z INFO:homeassistant.components.switch:Setting up switch.zha
2020-05-20T22:39:29.5896466Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=light>
2020-05-20T22:39:29.5897025Z INFO:homeassistant.setup:Setting up zone
2020-05-20T22:39:29.5897393Z INFO:homeassistant.components.light:Setting up light.zha
2020-05-20T22:39:29.5897878Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=zone, service=reload>
2020-05-20T22:39:29.5899457Z DEBUG:homeassistant.core:Bus:Handling <Event state_changed[L]: entity_id=zone.home, old_state=None, new_state=<state zone.home=zoning; hidden=True, latitude=32.87336, longitude=-117.22743, radius=100, passive=False, editable=True, friendly_name=test home, icon=mdi:home @ 2020-05-20T22:30:53.987299+00:00>>
2020-05-20T22:39:29.5900459Z INFO:homeassistant.setup:Setup of domain zone took 0.0 seconds.
2020-05-20T22:39:29.5900923Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=zone>
2020-05-20T22:39:29.5901379Z INFO:homeassistant.setup:Setting up device_tracker
2020-05-20T22:39:29.5901869Z DEBUG:homeassistant.core:Bus:Handling <Event service_registered[L]: domain=device_tracker, service=see>
2020-05-20T22:39:29.5902382Z INFO:homeassistant.setup:Setup of domain device_tracker took 0.0 seconds.
2020-05-20T22:39:29.5902886Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=device_tracker>
2020-05-20T22:39:29.5903406Z INFO:homeassistant.components.device_tracker:Setting up device_tracker.zha
2020-05-20T22:39:29.5903978Z DEBUG:homeassistant.core:Bus:Handling <Event device_registry_updated[L]: action=create, device_id=53dafeaaa8e8494d98f2f60cc42cf4f5>
2020-05-20T22:39:29.5905023Z DEBUG:homeassistant.components.zha.core.gateway:device - 47004:00:0d:6f:00:0a:90:69:e7 entering async_device_initialized - is_new_join: True
2020-05-20T22:39:29.5906040Z DEBUG:homeassistant.components.zha.core.gateway:device - 47004:00:0d:6f:00:0a:90:69:e7 has joined the ZHA zigbee network
2020-05-20T22:39:29.5906700Z DEBUG:homeassistant.components.zha.core.device:[47004](FakeModel): started configuration
2020-05-20T22:39:29.5907633Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:ZDO](FakeModel): 'async_configure' stage succeeded
2020-05-20T22:39:29.5908639Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0201]: bound 'thermostat' cluster: 0
2020-05-20T22:39:29.5910105Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0201]: attr reporting for '{'local_temp': (30, 900, 25), 'occupied_cooling_setpoint': (30, 900, 25), 'occupied_heating_setpoint': (30, 900, 25), 'unoccupied_cooling_setpoint': (30, 900, 25)}' on 'thermostat': 0
2020-05-20T22:39:29.5911875Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0201]: attr reporting for '{'unoccupied_heating_setpoint': (30, 900, 25), 'running_mode': (30, 900, 25), 'running_state': (30, 900, 5), 'system_mode': (30, 900, 25)}' on 'thermostat': 0
2020-05-20T22:39:29.5913456Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0201]: attr reporting for '{'occupancy': (30, 900, 1), 'pi_cooling_demand': (30, 900, 5), 'pi_heating_demand': (30, 900, 5)}' on 'thermostat': 0
2020-05-20T22:39:29.5914384Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0201]: finished channel configuration
2020-05-20T22:39:29.5915278Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0000]: bound 'basic' cluster: 0
2020-05-20T22:39:29.5915934Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0000]: finished channel configuration
2020-05-20T22:39:29.5916625Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0000]: initializing channel: from_cache: False
2020-05-20T22:39:29.5917539Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0019]: bound 'ota' cluster: 0
2020-05-20T22:39:29.5918191Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0019]: finished channel configuration
2020-05-20T22:39:29.5919087Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0201]: 'async_configure' stage succeeded
2020-05-20T22:39:29.5920150Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0000]: 'async_configure' stage succeeded
2020-05-20T22:39:29.5921052Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0019]: 'async_configure' stage succeeded
2020-05-20T22:39:29.5921709Z DEBUG:homeassistant.components.zha.core.device:[47004](FakeModel): completed configuration
2020-05-20T22:39:29.5922883Z DEBUG:homeassistant.components.zha.core.device:[47004](FakeModel): stored in registry: ZhaDeviceEntry(name='unk_manufacturer FakeModel', ieee='00:0d:6f:00:0a:90:69:e7', last_seen=1590013853.9415524)
2020-05-20T22:39:29.5924158Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0003]: executed 'trigger_effect' command with args: '(2, 0)' kwargs: '{}' result: [0]
2020-05-20T22:39:29.5924926Z DEBUG:homeassistant.components.zha.core.device:[47004](FakeModel): started initialization
2020-05-20T22:39:29.5925825Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:ZDO](FakeModel): 'async_initialize' stage succeeded
2020-05-20T22:39:29.5927223Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0201]: read attributes: Success: {'abs_min_heat_setpoint_limit': 800, 'abs_max_heat_setpoint_limit': 3000, 'abs_min_cool_setpoint_limit': 2000, 'abs_max_cool_setpoint_limit': 4000}. Failed: {}
2020-05-20T22:39:29.5928915Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0201]: read attributes: Success: {'max_cool_setpoint_limit': 3900, 'max_heat_setpoint_limit': 2900, 'min_cool_setpoint_limit': 2100, 'min_heat_setpoint_limit': 700}. Failed: {}
2020-05-20T22:39:29.5930609Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0201]: read attributes: Success: {'ctrl_seqe_of_oper': <ControlSequenceOfOperation.Cooling_and_Heating: 4>, 'local_temp': None, 'occupancy': 1, 'occupied_cooling_setpoint': 2500}. Failed: {}
2020-05-20T22:39:29.5932276Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0201]: read attributes: Success: {'occupied_heating_setpoint': 2200, 'pi_cooling_demand': None, 'pi_heating_demand': None, 'running_mode': <RunningMode.Off: 0>}. Failed: {}
2020-05-20T22:39:29.5933938Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0201]: read attributes: Success: {'running_state': None, 'system_mode': <SystemMode.Off: 0>, 'unoccupied_heating_setpoint': 2200, 'unoccupied_cooling_setpoint': 2300}. Failed: {}
2020-05-20T22:39:29.5934967Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0201]: initializing channel: from_cache: False
2020-05-20T22:39:29.5935672Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0000]: initializing channel: from_cache: False
2020-05-20T22:39:29.5936385Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0019]: initializing channel: from_cache: False
2020-05-20T22:39:29.5937259Z DEBUG:homeassistant.core:Bus:Handling <Event device_registry_updated[L]: action=update, device_id=53dafeaaa8e8494d98f2f60cc42cf4f5>
2020-05-20T22:39:29.5938036Z INFO:homeassistant.helpers.entity_registry:Registered new climate.zha entity: climate.unk_manufacturer_fakemodel_e769900a_thermostat
2020-05-20T22:39:29.5938889Z DEBUG:homeassistant.core:Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=climate.unk_manufacturer_fakemodel_e769900a_thermostat>
2020-05-20T22:39:29.5939903Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0201]: 'async_initialize' stage succeeded
2020-05-20T22:39:29.5940808Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0000]: 'async_initialize' stage succeeded
2020-05-20T22:39:29.5941720Z DEBUG:homeassistant.components.zha.core.channels.base:[47004:1:0x0019]: 'async_initialize' stage succeeded
2020-05-20T22:39:29.5942393Z DEBUG:homeassistant.components.zha.core.device:[47004](FakeModel): power source: Battery or Unknown
2020-05-20T22:39:29.5943010Z DEBUG:homeassistant.components.zha.core.device:[47004](FakeModel): completed initialization
2020-05-20T22:39:29.5943905Z DEBUG:homeassistant.helpers.restore_state:Not creating cache - no saved states found
2020-05-20T22:39:29.5944516Z DEBUG:homeassistant.helpers.restore_state:Dumping states
2020-05-20T22:39:29.5945340Z INFO:tests.common:Writing data to core.restore_state: {'version': 1, 'key': 'core.restore_state', 'data': []}
2020-05-20T22:39:29.5947296Z DEBUG:homeassistant.core:Bus:Handling <Event state_changed[L]: entity_id=climate.unk_manufacturer_fakemodel_e769900a_thermostat, old_state=None, new_state=<state climate.unk_manufacturer_fakemodel_e769900a_thermostat=unavailable; hvac_modes=('off', 'heat_cool', 'cool', 'heat'), min_temp=7.0, max_temp=39.0, friendly_name=unk_manufacturer FakeModel e769900a thermostat, supported_features=3 @ 2020-05-20T22:30:54.003814+00:00>>
2020-05-20T22:39:29.5948746Z DEBUG:homeassistant.components.zha.entity:climate.unk_manufacturer_fakemodel_e769900a_thermostat: Running mode: RunningMode.Off
2020-05-20T22:39:29.5949518Z DEBUG:homeassistant.components.zha.entity:climate.unk_manufacturer_fakemodel_e769900a_thermostat: Running state: None
2020-05-20T22:39:29.5952910Z DEBUG:homeassistant.core:Bus:Handling <Event state_changed[L]: entity_id=climate.unk_manufacturer_fakemodel_e769900a_thermostat, old_state=<state climate.unk_manufacturer_fakemodel_e769900a_thermostat=unavailable; hvac_modes=('off', 'heat_cool', 'cool', 'heat'), min_temp=7.0, max_temp=39.0, friendly_name=unk_manufacturer FakeModel e769900a thermostat, supported_features=3 @ 2020-05-20T22:30:54.003814+00:00>, new_state=<state climate.unk_manufacturer_fakemodel_e769900a_thermostat=off; hvac_modes=('off', 'heat_cool', 'cool', 'heat'), min_temp=7.0, max_temp=39.0, current_temperature=None, temperature=None, target_temp_high=None, target_temp_low=None, system_mode=[0]/off, occupancy=1, occupied_cooling_setpoint=2500, occupied_heating_setpoint=2200, unoccupied_heating_setpoint=2300, unoccupied_cooling_setpoint=2200, friendly_name=unk_manufacturer FakeModel e769900a thermostat, supported_features=3 @ 2020-05-20T22:30:54.004717+00:00>>
2020-05-20T22:39:29.5955467Z ------------------------------ Captured log setup ------------------------------
2020-05-20T22:39:29.5956022Z DEBUG    asyncio:selector_events.py:64 Using selector: EpollSelector
2020-05-20T22:39:29.5956529Z DEBUG    zigpy.group:group.py:98 Adding group: 4097, fixture group
2020-05-20T22:39:29.5957092Z INFO     homeassistant.loader:loader.py:208 Loaded test_package from custom_components.test_package
2020-05-20T22:39:29.5957682Z INFO     homeassistant.loader:loader.py:208 Loaded test from custom_components.test
2020-05-20T22:39:29.5958237Z INFO     homeassistant.loader:loader.py:208 Loaded zha from homeassistant.components.zha
2020-05-20T22:39:29.5958767Z INFO     homeassistant.setup:setup.py:154 Setting up zha
2020-05-20T22:39:29.5959271Z INFO     homeassistant.setup:setup.py:188 Setup of domain zha took 0.0 seconds.
2020-05-20T22:39:29.5960235Z DEBUG    homeassistant.components.zha.core.discovery:discovery.py:206 Group: fixture group:0x1001 has less than 2 members - skipping entity discovery
2020-05-20T22:39:29.5961101Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event device_registry_updated[L]: action=create, device_id=cba5b3729cef47aab2db88369c900597>
2020-05-20T22:39:29.5961865Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=zha, service=permit>
2020-05-20T22:39:29.5962543Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=zha, service=remove>
2020-05-20T22:39:29.5964296Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=zha, service=set_zigbee_cluster_attribute>
2020-05-20T22:39:29.5965089Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=zha, service=issue_zigbee_cluster_command>
2020-05-20T22:39:29.5965843Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=zha, service=issue_zigbee_group_command>
2020-05-20T22:39:29.5966573Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=zha, service=warning_device_squawk>
2020-05-20T22:39:29.5967504Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=zha, service=warning_device_warn>
2020-05-20T22:39:29.5968191Z DEBUG    homeassistant.components.zha.core.gateway:gateway.py:187 Loading battery powered devices
2020-05-20T22:39:29.5968796Z DEBUG    homeassistant.components.zha.core.gateway:gateway.py:196 Loading mains powered devices
2020-05-20T22:39:29.5969427Z INFO     homeassistant.loader:loader.py:208 Loaded binary_sensor from homeassistant.components.binary_sensor
2020-05-20T22:39:29.5970071Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event component_loaded[L]: component=zha>
2020-05-20T22:39:29.5970621Z INFO     homeassistant.setup:setup.py:154 Setting up binary_sensor
2020-05-20T22:39:29.5971159Z INFO     homeassistant.setup:setup.py:188 Setup of domain binary_sensor took 0.0 seconds.
2020-05-20T22:39:29.5971789Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event component_loaded[L]: component=binary_sensor>
2020-05-20T22:39:29.5972434Z INFO     homeassistant.components.binary_sensor:entity_platform.py:165 Setting up binary_sensor.zha
2020-05-20T22:39:29.5973050Z INFO     homeassistant.loader:loader.py:208 Loaded climate from homeassistant.components.climate
2020-05-20T22:39:29.5973643Z INFO     homeassistant.loader:loader.py:208 Loaded cover from homeassistant.components.cover
2020-05-20T22:39:29.5974244Z INFO     homeassistant.loader:loader.py:208 Loaded device_tracker from homeassistant.components.device_tracker
2020-05-20T22:39:29.5974856Z INFO     homeassistant.loader:loader.py:208 Loaded fan from homeassistant.components.fan
2020-05-20T22:39:29.5975437Z INFO     homeassistant.loader:loader.py:208 Loaded light from homeassistant.components.light
2020-05-20T22:39:29.5976006Z INFO     homeassistant.loader:loader.py:208 Loaded lock from homeassistant.components.lock
2020-05-20T22:39:29.5976716Z INFO     homeassistant.loader:loader.py:208 Loaded sensor from homeassistant.components.sensor
2020-05-20T22:39:29.5977323Z INFO     homeassistant.loader:loader.py:208 Loaded switch from homeassistant.components.switch
2020-05-20T22:39:29.5977913Z INFO     homeassistant.setup:setup.py:154 Setting up climate
2020-05-20T22:39:29.5978459Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=climate, service=turn_on>
2020-05-20T22:39:29.5979390Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=climate, service=turn_off>
2020-05-20T22:39:29.5980093Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=climate, service=set_hvac_mode>
2020-05-20T22:39:29.5980829Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=climate, service=set_preset_mode>
2020-05-20T22:39:29.5981559Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=climate, service=set_aux_heat>
2020-05-20T22:39:29.5982296Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=climate, service=set_temperature>
2020-05-20T22:39:29.5983015Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=climate, service=set_humidity>
2020-05-20T22:39:29.5983738Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=climate, service=set_fan_mode>
2020-05-20T22:39:29.5984456Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=climate, service=set_swing_mode>
2020-05-20T22:39:29.5985085Z INFO     homeassistant.setup:setup.py:188 Setup of domain climate took 0.0 seconds.
2020-05-20T22:39:29.5985606Z INFO     homeassistant.setup:setup.py:154 Setting up cover
2020-05-20T22:39:29.5986201Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=cover, service=open_cover>
2020-05-20T22:39:29.5986900Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=cover, service=close_cover>
2020-05-20T22:39:29.5987759Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=cover, service=set_cover_position>
2020-05-20T22:39:29.5988489Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=cover, service=stop_cover>
2020-05-20T22:39:29.5989178Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=cover, service=toggle>
2020-05-20T22:39:29.5989888Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=cover, service=open_cover_tilt>
2020-05-20T22:39:29.5990615Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=cover, service=close_cover_tilt>
2020-05-20T22:39:29.5991330Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=cover, service=stop_cover_tilt>
2020-05-20T22:39:29.5992071Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=cover, service=set_cover_tilt_position>
2020-05-20T22:39:29.5992817Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=cover, service=toggle_cover_tilt>
2020-05-20T22:39:29.5993455Z INFO     homeassistant.setup:setup.py:188 Setup of domain cover took 0.0 seconds.
2020-05-20T22:39:29.5994010Z INFO     homeassistant.loader:loader.py:208 Loaded zone from homeassistant.components.zone
2020-05-20T22:39:29.5994536Z INFO     homeassistant.setup:setup.py:154 Setting up fan
2020-05-20T22:39:29.5995118Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=fan, service=turn_on>
2020-05-20T22:39:29.5995801Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=fan, service=turn_off>
2020-05-20T22:39:29.5996493Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=fan, service=toggle>
2020-05-20T22:39:29.5997191Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=fan, service=set_speed>
2020-05-20T22:39:29.5997877Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=fan, service=oscillate>
2020-05-20T22:39:29.5998587Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=fan, service=set_direction>
2020-05-20T22:39:29.6001338Z INFO     homeassistant.setup:setup.py:188 Setup of domain fan took 0.0 seconds.
2020-05-20T22:39:29.6001919Z INFO     homeassistant.setup:setup.py:154 Setting up light
2020-05-20T22:39:29.6002403Z INFO     homeassistant.setup:setup.py:154 Setting up lock
2020-05-20T22:39:29.6002989Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=lock, service=unlock>
2020-05-20T22:39:29.6003681Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=lock, service=lock>
2020-05-20T22:39:29.6004370Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=lock, service=open>
2020-05-20T22:39:29.6004988Z INFO     homeassistant.setup:setup.py:188 Setup of domain lock took 0.0 seconds.
2020-05-20T22:39:29.6005553Z INFO     homeassistant.setup:setup.py:154 Setting up sensor
2020-05-20T22:39:29.6006251Z INFO     homeassistant.setup:setup.py:188 Setup of domain sensor took 0.0 seconds.
2020-05-20T22:39:29.6006787Z INFO     homeassistant.setup:setup.py:154 Setting up switch
2020-05-20T22:39:29.6007413Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=switch, service=turn_off>
2020-05-20T22:39:29.6008130Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=switch, service=turn_on>
2020-05-20T22:39:29.6008866Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=switch, service=toggle>
2020-05-20T22:39:29.6009480Z INFO     homeassistant.setup:setup.py:188 Setup of domain switch took 0.0 seconds.
2020-05-20T22:39:29.6010256Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event component_loaded[L]: component=climate>
2020-05-20T22:39:29.6010905Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event component_loaded[L]: component=cover>
2020-05-20T22:39:29.6011528Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event component_loaded[L]: component=fan>
2020-05-20T22:39:29.6012161Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event component_loaded[L]: component=lock>
2020-05-20T22:39:29.6012806Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event component_loaded[L]: component=sensor>
2020-05-20T22:39:29.6013430Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event component_loaded[L]: component=switch>
2020-05-20T22:39:29.6014427Z INFO     homeassistant.components.climate:entity_platform.py:165 Setting up climate.zha
2020-05-20T22:39:29.6015037Z INFO     homeassistant.components.cover:entity_platform.py:165 Setting up cover.zha
2020-05-20T22:39:29.6015883Z INFO     homeassistant.components.fan:entity_platform.py:165 Setting up fan.zha
2020-05-20T22:39:29.6017362Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=light, service=turn_on>
2020-05-20T22:39:29.6018079Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=light, service=turn_off>
2020-05-20T22:39:29.6018716Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=light, service=toggle>
2020-05-20T22:39:29.6019289Z INFO     homeassistant.setup:setup.py:188 Setup of domain light took 0.0 seconds.
2020-05-20T22:39:29.6020159Z INFO     homeassistant.components.lock:entity_platform.py:165 Setting up lock.zha
2020-05-20T22:39:29.6020740Z INFO     homeassistant.components.sensor:entity_platform.py:165 Setting up sensor.zha
2020-05-20T22:39:29.6021321Z INFO     homeassistant.components.switch:entity_platform.py:165 Setting up switch.zha
2020-05-20T22:39:29.6021931Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event component_loaded[L]: component=light>
2020-05-20T22:39:29.6022484Z INFO     homeassistant.setup:setup.py:154 Setting up zone
2020-05-20T22:39:29.6023009Z INFO     homeassistant.components.light:entity_platform.py:165 Setting up light.zha
2020-05-20T22:39:29.6023646Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=zone, service=reload>
2020-05-20T22:39:29.6026267Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event state_changed[L]: entity_id=zone.home, old_state=None, new_state=<state zone.home=zoning; hidden=True, latitude=32.87336, longitude=-117.22743, radius=100, passive=False, editable=True, friendly_name=test home, icon=mdi:home @ 2020-05-20T22:30:53.987299+00:00>>
2020-05-20T22:39:29.6027898Z INFO     homeassistant.setup:setup.py:188 Setup of domain zone took 0.0 seconds.
2020-05-20T22:39:29.6028868Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event component_loaded[L]: component=zone>
2020-05-20T22:39:29.6029466Z INFO     homeassistant.setup:setup.py:154 Setting up device_tracker
2020-05-20T22:39:29.6030069Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event service_registered[L]: domain=device_tracker, service=see>
2020-05-20T22:39:29.6030725Z INFO     homeassistant.setup:setup.py:188 Setup of domain device_tracker took 0.0 seconds.
2020-05-20T22:39:29.6031361Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event component_loaded[L]: component=device_tracker>
2020-05-20T22:39:29.6032009Z INFO     homeassistant.components.device_tracker:entity_platform.py:165 Setting up device_tracker.zha
2020-05-20T22:39:29.6032747Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event device_registry_updated[L]: action=create, device_id=53dafeaaa8e8494d98f2f60cc42cf4f5>
2020-05-20T22:39:29.6033969Z DEBUG    homeassistant.components.zha.core.gateway:gateway.py:514 device - 47004:00:0d:6f:00:0a:90:69:e7 entering async_device_initialized - is_new_join: True
2020-05-20T22:39:29.6036511Z DEBUG    homeassistant.components.zha.core.gateway:gateway.py:530 device - 47004:00:0d:6f:00:0a:90:69:e7 has joined the ZHA zigbee network
2020-05-20T22:39:29.6037401Z DEBUG    homeassistant.components.zha.core.device:device.py:683 [47004](FakeModel): started configuration
2020-05-20T22:39:29.6038406Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:338 [47004:ZDO](FakeModel): 'async_configure' stage succeeded
2020-05-20T22:39:29.6039427Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0201]: bound 'thermostat' cluster: 0
2020-05-20T22:39:29.6040962Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0201]: attr reporting for '{'local_temp': (30, 900, 25), 'occupied_cooling_setpoint': (30, 900, 25), 'occupied_heating_setpoint': (30, 900, 25), 'unoccupied_cooling_setpoint': (30, 900, 25)}' on 'thermostat': 0
2020-05-20T22:39:29.6042801Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0201]: attr reporting for '{'unoccupied_heating_setpoint': (30, 900, 25), 'running_mode': (30, 900, 25), 'running_state': (30, 900, 5), 'system_mode': (30, 900, 25)}' on 'thermostat': 0
2020-05-20T22:39:29.6044484Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0201]: attr reporting for '{'occupancy': (30, 900, 1), 'pi_cooling_demand': (30, 900, 5), 'pi_heating_demand': (30, 900, 5)}' on 'thermostat': 0
2020-05-20T22:39:29.6045468Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0201]: finished channel configuration
2020-05-20T22:39:29.6046441Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0000]: bound 'basic' cluster: 0
2020-05-20T22:39:29.6047167Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0000]: finished channel configuration
2020-05-20T22:39:29.6047921Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0000]: initializing channel: from_cache: False
2020-05-20T22:39:29.6048915Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0019]: bound 'ota' cluster: 0
2020-05-20T22:39:29.6049636Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0019]: finished channel configuration
2020-05-20T22:39:29.6050697Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0201]: 'async_configure' stage succeeded
2020-05-20T22:39:29.6051690Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0000]: 'async_configure' stage succeeded
2020-05-20T22:39:29.6052657Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0019]: 'async_configure' stage succeeded
2020-05-20T22:39:29.6053377Z DEBUG    homeassistant.components.zha.core.device:device.py:683 [47004](FakeModel): completed configuration
2020-05-20T22:39:29.6054639Z DEBUG    homeassistant.components.zha.core.device:device.py:683 [47004](FakeModel): stored in registry: ZhaDeviceEntry(name='unk_manufacturer FakeModel', ieee='00:0d:6f:00:0a:90:69:e7', last_seen=1590013853.9415524)
2020-05-20T22:39:29.6056025Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0003]: executed 'trigger_effect' command with args: '(2, 0)' kwargs: '{}' result: [0]
2020-05-20T22:39:29.6056974Z DEBUG    homeassistant.components.zha.core.device:device.py:683 [47004](FakeModel): started initialization
2020-05-20T22:39:29.6057971Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:338 [47004:ZDO](FakeModel): 'async_initialize' stage succeeded
2020-05-20T22:39:29.6059460Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0201]: read attributes: Success: {'abs_min_heat_setpoint_limit': 800, 'abs_max_heat_setpoint_limit': 3000, 'abs_min_cool_setpoint_limit': 2000, 'abs_max_cool_setpoint_limit': 4000}. Failed: {}
2020-05-20T22:39:29.6061218Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0201]: read attributes: Success: {'max_cool_setpoint_limit': 3900, 'max_heat_setpoint_limit': 2900, 'min_cool_setpoint_limit': 2100, 'min_heat_setpoint_limit': 700}. Failed: {}
2020-05-20T22:39:29.6063130Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0201]: read attributes: Success: {'ctrl_seqe_of_oper': <ControlSequenceOfOperation.Cooling_and_Heating: 4>, 'local_temp': None, 'occupancy': 1, 'occupied_cooling_setpoint': 2500}. Failed: {}
2020-05-20T22:39:29.6064881Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0201]: read attributes: Success: {'occupied_heating_setpoint': 2200, 'pi_cooling_demand': None, 'pi_heating_demand': None, 'running_mode': <RunningMode.Off: 0>}. Failed: {}
2020-05-20T22:39:29.6068126Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0201]: read attributes: Success: {'running_state': None, 'system_mode': <SystemMode.Off: 0>, 'unoccupied_heating_setpoint': 2200, 'unoccupied_cooling_setpoint': 2300}. Failed: {}
2020-05-20T22:39:29.6069318Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0201]: initializing channel: from_cache: False
2020-05-20T22:39:29.6070114Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0000]: initializing channel: from_cache: False
2020-05-20T22:39:29.6070879Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0019]: initializing channel: from_cache: False
2020-05-20T22:39:29.6071678Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event device_registry_updated[L]: action=update, device_id=53dafeaaa8e8494d98f2f60cc42cf4f5>
2020-05-20T22:39:29.6072527Z INFO     homeassistant.helpers.entity_registry:entity_registry.py:275 Registered new climate.zha entity: climate.unk_manufacturer_fakemodel_e769900a_thermostat
2020-05-20T22:39:29.6073396Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=climate.unk_manufacturer_fakemodel_e769900a_thermostat>
2020-05-20T22:39:29.6074557Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0201]: 'async_initialize' stage succeeded
2020-05-20T22:39:29.6075562Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0000]: 'async_initialize' stage succeeded
2020-05-20T22:39:29.6076563Z DEBUG    homeassistant.components.zha.core.channels.base:base.py:280 [47004:1:0x0019]: 'async_initialize' stage succeeded
2020-05-20T22:39:29.6077293Z DEBUG    homeassistant.components.zha.core.device:device.py:683 [47004](FakeModel): power source: Battery or Unknown
2020-05-20T22:39:29.6078007Z DEBUG    homeassistant.components.zha.core.device:device.py:683 [47004](FakeModel): completed initialization
2020-05-20T22:39:29.6078917Z DEBUG    homeassistant.helpers.restore_state:restore_state.py:81 Not creating cache - no saved states found
2020-05-20T22:39:29.6079531Z DEBUG    homeassistant.helpers.restore_state:restore_state.py:159 Dumping states
2020-05-20T22:39:29.6080469Z INFO     tests.common:common.py:944 Writing data to core.restore_state: {'version': 1, 'key': 'core.restore_state', 'data': []}
2020-05-20T22:39:29.6082514Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event state_changed[L]: entity_id=climate.unk_manufacturer_fakemodel_e769900a_thermostat, old_state=None, new_state=<state climate.unk_manufacturer_fakemodel_e769900a_thermostat=unavailable; hvac_modes=('off', 'heat_cool', 'cool', 'heat'), min_temp=7.0, max_temp=39.0, friendly_name=unk_manufacturer FakeModel e769900a thermostat, supported_features=3 @ 2020-05-20T22:30:54.003814+00:00>>
2020-05-20T22:39:29.6084051Z DEBUG    homeassistant.components.zha.entity:entity.py:143 climate.unk_manufacturer_fakemodel_e769900a_thermostat: Running mode: RunningMode.Off
2020-05-20T22:39:29.6084901Z DEBUG    homeassistant.components.zha.entity:entity.py:143 climate.unk_manufacturer_fakemodel_e769900a_thermostat: Running state: None
2020-05-20T22:39:29.6088365Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event state_changed[L]: entity_id=climate.unk_manufacturer_fakemodel_e769900a_thermostat, old_state=<state climate.unk_manufacturer_fakemodel_e769900a_thermostat=unavailable; hvac_modes=('off', 'heat_cool', 'cool', 'heat'), min_temp=7.0, max_temp=39.0, friendly_name=unk_manufacturer FakeModel e769900a thermostat, supported_features=3 @ 2020-05-20T22:30:54.003814+00:00>, new_state=<state climate.unk_manufacturer_fakemodel_e769900a_thermostat=off; hvac_modes=('off', 'heat_cool', 'cool', 'heat'), min_temp=7.0, max_temp=39.0, current_temperature=None, temperature=None, target_temp_high=None, target_temp_low=None, system_mode=[0]/off, occupancy=1, occupied_cooling_setpoint=2500, occupied_heating_setpoint=2200, unoccupied_heating_setpoint=2300, unoccupied_cooling_setpoint=2200, friendly_name=unk_manufacturer FakeModel e769900a thermostat, supported_features=3 @ 2020-05-20T22:30:54.004717+00:00>>
2020-05-20T22:39:29.6091115Z ----------------------------- Captured stderr call -----------------------------
2020-05-20T22:39:29.6091864Z DEBUG:homeassistant.core:Bus:Handling <Event call_service[L]: domain=climate, service=set_fan_mode, service_data=entity_id=climate.unk_manufacturer_fakemodel_e769900a_thermostat, fan_mode=on>
2020-05-20T22:39:29.6092831Z ------------------------------ Captured log call -------------------------------
2020-05-20T22:39:29.6093612Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event call_service[L]: domain=climate, service=set_fan_mode, service_data=entity_id=climate.unk_manufacturer_fakemodel_e769900a_thermostat, fan_mode=on>
2020-05-20T22:39:29.6094600Z --------------------------- Captured stderr teardown ---------------------------
2020-05-20T22:39:29.6095109Z DEBUG:homeassistant.core:Bus:Handling <Event homeassistant_stop[L]>
2020-05-20T22:39:29.6095649Z DEBUG:homeassistant.components.zha.core.gateway:Shutting down ZHA ControllerApplication
2020-05-20T22:39:29.6096177Z DEBUG:homeassistant.helpers.restore_state:Dumping states
2020-05-20T22:39:29.6096833Z DEBUG:homeassistant.core:Bus:Handling <Event homeassistant_final_write[L]>
2020-05-20T22:39:29.6100631Z INFO:tests.common:Writing data to core.device_registry: {'version': 1, 'key': 'core.device_registry', 'data': {'devices': [{'config_entries': ['14bb6dd81ee544a394d85da396d97486'], 'connections': [('zigbee', "<MagicMock name='mock()' id='139977087276200'>")], 'identifiers': [('zha', "<MagicMock name='mock()' id='139977087276200'>")], 'manufacturer': 'ZHA', 'model': 'ESZP: HUSBZB-1, Elelabs, Telegesis, Silabs EmberZNet protocol', 'name': 'Zigbee Coordinator', 'sw_version': None, 'entry_type': None, 'id': 'cba5b3729cef47aab2db88369c900597', 'via_device_id': None, 'area_id': None, 'name_by_user': None}, {'config_entries': ['14bb6dd81ee544a394d85da396d97486'], 'connections': [('zigbee', '00:0d:6f:00:0a:90:69:e7')], 'identifiers': [('zha', '00:0d:6f:00:0a:90:69:e7')], 'manufacturer': 'unk_manufacturer', 'model': 'FakeModel', 'name': 'unk_manufacturer FakeModel', 'sw_version': None, 'entry_type': None, 'id': '53dafeaaa8e8494d98f2f60cc42cf4f5', 'via_device_id': 'cba5b3729cef47aab2db88369c900597', 'area_id': None, 'name_by_user': None}]}}
2020-05-20T22:39:29.6105448Z INFO:tests.common:Writing data to core.entity_registry: {'version': 1, 'key': 'core.entity_registry', 'data': {'entities': [{'entity_id': 'climate.unk_manufacturer_fakemodel_e769900a_thermostat', 'config_entry_id': '14bb6dd81ee544a394d85da396d97486', 'device_id': '53dafeaaa8e8494d98f2f60cc42cf4f5', 'unique_id': '00:0d:6f:00:0a:90:69:e7-1', 'platform': 'zha', 'name': None, 'icon': None, 'disabled_by': None, 'capabilities': {'hvac_modes': ('off', 'heat_cool', 'cool', 'heat'), 'min_temp': 7.0, 'max_temp': 39.0}, 'supported_features': 3, 'device_class': None, 'unit_of_measurement': None, 'original_name': 'unk_manufacturer FakeModel e769900a thermostat', 'original_icon': None}]}}
2020-05-20T22:39:29.6108033Z INFO:tests.common:Writing data to zha.storage: {'version': 1, 'key': 'zha.storage', 'data': {'devices': [{'name': 'unk_manufacturer FakeModel', 'ieee': '00:0d:6f:00:0a:90:69:e7', 'last_seen': 1590013853.9415524}]}}
2020-05-20T22:39:29.6112132Z INFO:tests.common:Writing data to core.restore_state: {'version': 1, 'key': 'core.restore_state', 'data': [{'state': {'entity_id': 'climate.unk_manufacturer_fakemodel_e769900a_thermostat', 'state': 'off', 'attributes': {'hvac_modes': ('off', 'heat_cool', 'cool', 'heat'), 'min_temp': 7.0, 'max_temp': 39.0, 'current_temperature': None, 'temperature': None, 'target_temp_high': None, 'target_temp_low': None, 'system_mode': '[0]/off', 'occupancy': 1, 'occupied_cooling_setpoint': 2500, 'occupied_heating_setpoint': 2200, 'unoccupied_heating_setpoint': 2300, 'unoccupied_cooling_setpoint': 2200, 'friendly_name': 'unk_manufacturer FakeModel e769900a thermostat', 'supported_features': 3}, 'last_changed': datetime.datetime(2020, 5, 20, 22, 30, 54, 4717, tzinfo=<UTC>), 'last_updated': datetime.datetime(2020, 5, 20, 22, 30, 54, 4717, tzinfo=<UTC>), 'context': {'id': 'c5034a435b214da5a538df205a2edd07', 'parent_id': None, 'user_id': None}}, 'last_seen': datetime.datetime(2020, 5, 20, 22, 30, 54, 49588, tzinfo=<UTC>)}]}
2020-05-20T22:39:29.6114812Z DEBUG:homeassistant.core:Bus:Handling <Event homeassistant_close[L]>
2020-05-20T22:39:29.6115575Z ---------------------------- Captured log teardown -----------------------------
2020-05-20T22:39:29.6116122Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event homeassistant_stop[L]>
2020-05-20T22:39:29.6116733Z DEBUG    homeassistant.components.zha.core.gateway:gateway.py:608 Shutting down ZHA ControllerApplication
2020-05-20T22:39:29.6117348Z DEBUG    homeassistant.helpers.restore_state:restore_state.py:159 Dumping states
2020-05-20T22:39:29.6117912Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event homeassistant_final_write[L]>
2020-05-20T22:39:29.6122184Z INFO     tests.common:common.py:944 Writing data to core.device_registry: {'version': 1, 'key': 'core.device_registry', 'data': {'devices': [{'config_entries': ['14bb6dd81ee544a394d85da396d97486'], 'connections': [('zigbee', "<MagicMock name='mock()' id='139977087276200'>")], 'identifiers': [('zha', "<MagicMock name='mock()' id='139977087276200'>")], 'manufacturer': 'ZHA', 'model': 'ESZP: HUSBZB-1, Elelabs, Telegesis, Silabs EmberZNet protocol', 'name': 'Zigbee Coordinator', 'sw_version': None, 'entry_type': None, 'id': 'cba5b3729cef47aab2db88369c900597', 'via_device_id': None, 'area_id': None, 'name_by_user': None}, {'config_entries': ['14bb6dd81ee544a394d85da396d97486'], 'connections': [('zigbee', '00:0d:6f:00:0a:90:69:e7')], 'identifiers': [('zha', '00:0d:6f:00:0a:90:69:e7')], 'manufacturer': 'unk_manufacturer', 'model': 'FakeModel', 'name': 'unk_manufacturer FakeModel', 'sw_version': None, 'entry_type': None, 'id': '53dafeaaa8e8494d98f2f60cc42cf4f5', 'via_device_id': 'cba5b3729cef47aab2db88369c900597', 'area_id': None, 'name_by_user': None}]}}
2020-05-20T22:39:29.6127162Z INFO     tests.common:common.py:944 Writing data to core.entity_registry: {'version': 1, 'key': 'core.entity_registry', 'data': {'entities': [{'entity_id': 'climate.unk_manufacturer_fakemodel_e769900a_thermostat', 'config_entry_id': '14bb6dd81ee544a394d85da396d97486', 'device_id': '53dafeaaa8e8494d98f2f60cc42cf4f5', 'unique_id': '00:0d:6f:00:0a:90:69:e7-1', 'platform': 'zha', 'name': None, 'icon': None, 'disabled_by': None, 'capabilities': {'hvac_modes': ('off', 'heat_cool', 'cool', 'heat'), 'min_temp': 7.0, 'max_temp': 39.0}, 'supported_features': 3, 'device_class': None, 'unit_of_measurement': None, 'original_name': 'unk_manufacturer FakeModel e769900a thermostat', 'original_icon': None}]}}
2020-05-20T22:39:29.6129820Z INFO     tests.common:common.py:944 Writing data to zha.storage: {'version': 1, 'key': 'zha.storage', 'data': {'devices': [{'name': 'unk_manufacturer FakeModel', 'ieee': '00:0d:6f:00:0a:90:69:e7', 'last_seen': 1590013853.9415524}]}}
2020-05-20T22:39:29.6133869Z INFO     tests.common:common.py:944 Writing data to core.restore_state: {'version': 1, 'key': 'core.restore_state', 'data': [{'state': {'entity_id': 'climate.unk_manufacturer_fakemodel_e769900a_thermostat', 'state': 'off', 'attributes': {'hvac_modes': ('off', 'heat_cool', 'cool', 'heat'), 'min_temp': 7.0, 'max_temp': 39.0, 'current_temperature': None, 'temperature': None, 'target_temp_high': None, 'target_temp_low': None, 'system_mode': '[0]/off', 'occupancy': 1, 'occupied_cooling_setpoint': 2500, 'occupied_heating_setpoint': 2200, 'unoccupied_heating_setpoint': 2300, 'unoccupied_cooling_setpoint': 2200, 'friendly_name': 'unk_manufacturer FakeModel e769900a thermostat', 'supported_features': 3}, 'last_changed': datetime.datetime(2020, 5, 20, 22, 30, 54, 4717, tzinfo=<UTC>), 'last_updated': datetime.datetime(2020, 5, 20, 22, 30, 54, 4717, tzinfo=<UTC>), 'context': {'id': 'c5034a435b214da5a538df205a2edd07', 'parent_id': None, 'user_id': None}}, 'last_seen': datetime.datetime(2020, 5, 20, 22, 30, 54, 49588, tzinfo=<UTC>)}]}
2020-05-20T22:39:29.6136799Z DEBUG    homeassistant.core:core.py:597 Bus:Handling <Event homeassistant_close[L]>
```

## Additional information

--- a/tests/components/zha/test_climate.py
+++ b/tests/components/zha/test_climate.py
@@ -1,5 +1,4 @@
 """Test zha climate."""
-import logging
 
 import pytest
 import zigpy.zcl.clusters
@@ -1001,34 +1000,32 @@ async def test_fan_mode(hass, device_climate_fan):
     assert state.attributes[ATTR_FAN_MODE] == FAN_ON
 
 
-async def test_set_fan_mode_no_fan(hass, device_climate, caplog):
+async def test_set_fan_mode_no_fan(hass, device_climate):
     """Test setting fan mode on fun less climate."""
 
     entity_id = await find_entity_id(DOMAIN, device_climate, hass)
 
-    with caplog.at_level(logging.DEBUG):
-        await hass.services.async_call(
-            DOMAIN,
-            SERVICE_SET_FAN_MODE,
-            {ATTR_ENTITY_ID: entity_id, ATTR_FAN_MODE: FAN_ON},
-            blocking=True,
-        )
-    assert "Fan is not supported" in caplog.text
+    await hass.services.async_call(
+        DOMAIN,
+        SERVICE_SET_FAN_MODE,
+        {ATTR_ENTITY_ID: entity_id, ATTR_FAN_MODE: FAN_ON},
+        blocking=True,
+    )
 
 
-async def test_set_fan_mode_not_supported(hass, device_climate_fan, caplog):
+async def test_set_fan_mode_not_supported(hass, device_climate_fan):
     """Test fan setting unsupported mode."""
 
     entity_id = await find_entity_id(DOMAIN, device_climate_fan, hass)
+    fan_cluster = device_climate_fan.device.endpoints[1].fan
 
-    with caplog.at_level(logging.DEBUG):
-        await hass.services.async_call(
-            DOMAIN,
-            SERVICE_SET_FAN_MODE,
-            {ATTR_ENTITY_ID: entity_id, ATTR_FAN_MODE: FAN_LOW},
-            blocking=True,
-        )
-    assert "Unsupported 'low' fan mode" in caplog.text
+    await hass.services.async_call(
+        DOMAIN,
+        SERVICE_SET_FAN_MODE,
+        {ATTR_ENTITY_ID: entity_id, ATTR_FAN_MODE: FAN_LOW},
+        blocking=True,
+    )
+    assert fan_cluster.write_attributes.await_count == 0
 
 
 async def test_set_fan_mode(hass, device_climate_fan):

